tests/common/platform/device_utils.py

```diff
@@ -45,7 +45,7 @@ def get_dut_psu_line_pattern(dut):
             PSU 2  N/A      N/A               12.01           4.12        49.50  OK        green
 
         """
-        psu_line_pattern = re.compile(r"PSU\s+(\d+).*?(OK|NOT OK|NOT PRESENT)\s+(green|amber|red|off)")
+        psu_line_pattern = re.compile(r"PSU\s+(\d+).*?(OK|NOT OK|NOT PRESENT|WARNING)\s+(green|amber|red|off)")
```

Comment:
Do you need change the comment of this function as the output of `show platform psu` is changed

Reply:
There were 3 statues in the output of `status` but only `OK` was in the example.
So I do not tend to add all the statues.

Reply:
I remember you introduced new columns, isn't it?

Reply:
yes, but it's in `psuutil` but this one is used to parse `show platform psustatus` which is `psushow`

Reply:
got it